Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ReTestItems"
 uuid = "817f1d60-ba6b-4fd5-9520-3cf149f6a823"
-version = "1.23.3"
+version = "1.24.0"
```

Comment:
As I'll be adding backtraces also once this merges, maybe let's leave the version bump for the next PR?

Reply:
I agree :+1: